astro/transform.py

```diff
@@ -231,4 +231,90 @@ def cartesian_to_classical(self, r, v):
             w = float('nan')
             TA = float('nan')
 
-        return a, e, i, node, w, TA
\ No newline at end of file
+        return a, e, i, node, w, TA
+
+    def ra_dec_to_cartesian(self, r_mag, ra, dec, dr_mag, dra, ddec):
+        """ Convert right ascension and declination to cartesian elements
+
+        Args:
+            r_mag (float) - radius of object
+            ra (float) - right ascension (rad)
+            dec (float) - declination (rad)
+            dr_mag (float) - radius rate of object
+            dra (float) - right ascension rate (rad/s)
+            ddec (float) - right declination rate (rad/s)
+
+        Returns:
+            r (numpy array) - cartesian position
+            v (numpy array) - cartesian velocity
+        """
+
+        # Get position vector
+        r = np.array([0.0] * 3)
+        r[0] = r_mag * cos(dec) * cos(ra)
+        r[1] = r_mag * cos(dec) * sin(ra)
+        r[2] = r_mag * sin(dec)
+
+        # Get velocity vector
```

Comment:
I haven't done the velocity transform before, so I looked it up... but the signs seem different than this reference:
http://www.astrosurf.com/jephem/library/li110spherCart_en.htm

Maybe there is more than one sign combination that works?

Which reference is this from ?

Reply:
Thanks for taking a look! I'm using Vallado and can double-check the signs